src/main/scala/akka/persistence/cassandra/journal/PubSubThrottler.scala

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (C) 2016 Typesafe Inc. <http://www.typesafe.com>
+ */
+package akka.persistence.cassandra.journal
+
+import scala.concurrent.duration.FiniteDuration
+
+import akka.actor.{ Actor, ActorRef }
+
+/**
+ * Proxies messages to another actor, only allowing identical messages through once every [interval].
+ */
+class PubSubThrottler(delegate: ActorRef, interval: FiniteDuration) extends Actor {
+  import PubSubThrottler._
+  import context.dispatcher
+
+  val intervalMs = interval.toMillis
+  var seen = Map.empty[Any, Long]
+  context.system.scheduler.schedule(interval, interval, self, Cleanup)
+
+  def receive = {
+    case Cleanup =>
+      seen = seen.filterNot { case (msg, time) => expired(time) }
+
+    case msg =>
+      if (seen.get(msg).filterNot(expired).isEmpty) {
+        // Not seen yet, or seen more than [interval] ago
+        delegate forward msg
+      }
+      seen += msg -> System.currentTimeMillis
+
+  }
+
+  private def expired(time: Long): Boolean = time + intervalMs < System.currentTimeMillis
```

Comment:
I thought this was exactly for difference measurement. It's expired after a duration. Anyway, it should not be needed.